### PR TITLE
Pipelines-RN-1.12.2 Release Notes for Pipelines 1.12.2

### PR DESCRIPTION
**Pull request by eromanova97**
• **Aligned team**: Dev Tools
• **Please cherry-pick to**: `pipelines-docs-1.12`
• **Preview page**: [Release notes for Red Hat OpenShift Pipelines General Availability 1.12.2](https://67276--docspreview.netlify.app/openshift-pipelines/latest/about/op-release-notes#release-notes-1-12-2_op-release-notes)
• **SME Review**: Completed by @rupalibehera, @chmouel 
• **QE review**: Completed by @VeereshAradhya

### CODE REVIEW

File: modules/op-release-notes-1-12.adoc [hunk 1]
```diff
@@ -202,3 +202,15 @@ With this update, {pipelines-title} General Availability (GA) 1.12.1 is availabl
...
+
+[id="release-notes-1-12-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.12.2
+
+With this update, {pipelines-title} General Availability (GA) 1.12.2 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-12-2_{context}"]
+=== Fixed issues
+
+* Before this update, the GitHub `basic-auth` secret for the latest pipeline run would get deleted when the `max-keep-runs` parameter was exceeded. With this update, the GitHub `basic-auth` secret no longer gets deleted and it still exists on the cluster if its owner pipeline run also exists on the lcuster.
```
Comment (chmouel) [inline modules/op-release-notes-1-12.adoc:214]:
```suggestion
* Before this update, the generated git secret for the latest pipeline run would get deleted when the `max-keep-runs` parameter was exceeded. With this update, the git secret  no longer gets deleted on the latest PipelineRun
```

it's not specific to Github and it's not a basic-auth secret

**Review by rupalibehera — APPROVED** [review]

**Review by VeereshAradhya — APPROVED** [review]

File: modules/op-release-notes-1-12.adoc [hunk 2]
```diff
@@ -202,3 +202,15 @@ With this update, {pipelines-title} General Availability (GA) 1.12.1 is availabl
...
+
+[id="release-notes-1-12-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.12.2
+
+With this update, {pipelines-title} General Availability (GA) 1.12.2 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-12-2_{context}"]
+=== Fixed issues
+
+* Before this update, the generated git secret for the latest pipeline run would get deleted when the `max-keep-runs` parameter was exceeded. With this update, the git secret no longer gets deleted on the latest pipeline run.
```
Comment (abrennan89) [inline modules/op-release-notes-1-12.adoc:214]:
```suggestion
* Before this update, the generated Git secret for the latest pipeline run was deleted when the `max-keep-runs` parameter was exceeded. With this update, the Git secret is no longer deleted on the latest pipeline run.
```

Per https://redhat-documentation.github.io/supplementary-style-guide/#git Git should be capitalized. Also, suggested editing for tenses.

**Review by abrennan89 — COMMENTED** [review]
One comment before merging @ekristova

**Comment by ekristova** [comment]
@abrennan89 I made the suggested changes. Thank you!

**Pull request merged**